controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3448,8 +3448,8 @@ XMLStreamException discoveryOptionsMustBeDeclared(String adminOnlyCmd, String po
 
     // ---- Credential Store related messages -----------
 
-    @Message(id = 423, value = "Name of the credential store has to be specified in this credential-reference")
-    IllegalStateException nameOfCredentialStoreHasToBeSpecified();
+    @Message(id = 423, value = "Masked password command has the wrong format.\nUsage: MASK-<encoded secret>;<salt>;<iteration count> where <salt>=UTF-8 characters, <iteration count>=reasonable sized positive integer")
```

Comment:
@jamezp is the "\n" here the correct syntax?

Granted, I'm sure we have lots of other places we use it.